seqr/views/utils/orm_to_json_utils.py

```diff
@@ -757,10 +757,27 @@ def get_json_for_matchmaker_submission(submission):
         additional_model_fields=['contact_name', 'contact_href', 'submission_id'])
 
 
-def get_json_for_rna_seq_outliers(models, **kwargs):
-    additional_values = {
+def get_json_for_rna_seq_outliers(models, max_significant_num_per_tissue=None, **kwargs):
+    significant_field = models.model.SIGNIFICANCE_FIELD
+    significant_threshold = models.model.SIGNIFICANCE_THRESHOLD
+    additional_values = None if max_significant_num_per_tissue else {
```

Comment:
It will be cleaner to just do the assignment the same for both data types, the performance optimization you are getting for the outlier models is pretty small relative to the code complexity thats being added